packages/slate/src/utils/path-utils.js

```diff
@@ -102,7 +131,7 @@ function increment(path, n = 1, index = path.size - 1) {
 
 function isAbove(path, target) {
   const [p, t] = crop(path, target)
-  return path.size < target.size && compare(p, t) === 0
+  return path.size < target.size && compareOfSameSize(p, t) === 0
```

Comment:
I think we can use the new compare here?

Reply:
:+1: